src/api/routes/user.routes.ts

```diff
@@ -4,6 +4,11 @@ import * as controller from "../controllers/user.controller";
 
 const userRouter = express.Router();
 
+// Create User 
 userRouter.post("/", hashPassword, controller.createUserController);
```

Comment:
This should probably be in the auth router. Since we want it in the /api/auth/signup endpoint